lib/Conversation/index.js

```diff
@@ -15,6 +16,7 @@ class Conversation extends Component {
       activateConversation: PropTypes.func.isRequired,
```

Comment:
I can't comment on the specific line here, but I believe `id` should also have an `isRequired` to its propTypes. If there's a conversation being created, an ID truly is required for everything else to work :)